search-app/src/cmr/search/results_handlers/umm_json_results_handler.clj

```diff
@@ -36,7 +38,9 @@
 
 (defmethod elastic-search-index/concept-type+result-format->fields [:collection :umm-json-results]
   [concept-type query]
-  results-helper/meta-fields)
+  (concat
+   results-helper/meta-fields
+   ["s3-bucket-and-object-prefix-names"]))
```

Comment:
Why is this ["s3-bucket-and-object-prefix-names"] hardcoded here and not included with the other fields that are defined in the meta-fields list?  Please put s3-bucket-and-object-prefix-names into the meta-fields list.  The documentation in the meta-fields function states: "Defines the fields in elastic search we retrieve to populate the meta fields in the response." and s3-bucket-and-object-prefix-names is a field in elastic search we retrieve and it is being put into the meta fields response section.